packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts

```diff
@@ -44,6 +51,13 @@ export class LogContext {
   private readonly launchArgs: LaunchRequestArguments;
   private readonly logLines: string[] = [];
   private state: DebugLogState | undefined;
+  private frameHandles = new Handles<ApexDebugStackFrameInfo>();
+  private scopeHandles = new Handles<ScopeContainer>();
+  private staticVariablesClassMap = new Map<
```

Comment:
Static variables should be shared amongst frames of the same class. This map will store a variable->variableName map to every class name/namespaced used.